.github/actions/deploy-hyp3/action.yml

```
@@ -59,8 +59,8 @@ inputs:
   PERMISSIONS_BOUNDARY_ARN:
     description: "Arn of permissions boundary to provide to any created IAM roles"
     required: true
-  DEPLOY_LAMBDAS_IN_VPC:
-    description: "Whether or not to deploy lambda functions in a VPC"
+  EARTHDATA_CLOUD:
+    description: "Whether or not to exclude resources/configurations incompatable with Earthdata Cloud"
```

We're excluding the bucket policy, but deploying the lambdas into the VPC feels like adding rather than removing. Maybe
```suggestion
    description: "Modify resources/configurations to be compatible with Earthdata Cloud"
```